sleap_io/io/slp.py

```
@@ -91,23 +81,44 @@
+            # This is an ImageVideo.
+            # TODO: Path resolution.
+            video_path = backend_metadata["filenames"]
+
+        try:
+            backend = VideoBackend.from_filename(
+                video_path,
+                dataset=backend_metadata.get("dataset", None),
+                grayscale=backend_metadata.get("grayscale", None),
+                input_format=backend_metadata.get("input_format", None),
+            )
+        except:
```

Ruff

Do not use bare `except`

(E722)